lib/bolt/config.rb

```diff
@@ -497,6 +497,25 @@ def validate
       unless TRANSPORT_CONFIG.include?(transport)
         raise UnknownTransportError, transport
       end
+
+      # Warn the user how they should be using the 'puppetfile' or
+      # 'module-install' config options. We don't error here since these
+      # settings can be set at the user or system level.
+      if @project.modules && puppetfile_config.any? && module_install.empty?
+        command = Bolt::Util.powershell? ? 'bolt project migrate' : 'Update-BoltProject'
```

Comment:
I think these are backwards

Reply:
But what if I _want_ to confuse people? :P Fixed.